### DISCUSSION TITLE
strelka custom local rules

### DISCUSSION THREAD

**Discussion by sinnetll19** [Security-Onion-Solutions/securityonion#3347] [Unsupported Versions]
i followed the instructions on security onion 2 documentation to add local yara rules to strelka. after running so-strelka-restart everything comes back normal. how can i verify that strelka is in fact using my added local rules? is there a file where all the rules are located similar to suricata all.rules ?

**Comment by palichx** [comment]
hi. 
as i understand  yara have no one file like all.rules.
you can find actual rules on sensor in /opt/so/conf/strelka/rules/**localrules**

**Comment by weslambert** [comment]
Currently, Strelka globs all the files referenced by the YARA path. If a rule fails, the entire package fails for every scan.  One can assume that if YARA works for one scan, all rules should be successfully loaded.  There is not really any other verification or tracking at the moment, but we might consider enhancing this in the future.  You could also try using `plyara` to provide a check for YARA compatibility prior to loading the new rules.  Maybe we can consider doing something similar in the Strelka state, or using plyara in a standalone manner.
